src/style.css

```
@@ -0,0 +1,39 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  margin-left: 4px;
+  width: 16px;
+  height: 16px;
+  background: url("images/star.svg") center center no-repeat;
+}
+
+.stars__star:first-child {
+  margin-left: 0;
+}
```

use :not(last-child) to add margin
```suggestion
.stars__star {
  width: 16px;
  height: 16px;
  background: url("images/star.svg") center center no-repeat;
}

.stars__star:not(first-child) {
  margin-left: 4px;
}
```